packages/dev/package.json

```diff
@@ -9,11 +9,11 @@
     "serve": "serve public"
   },
   "dependencies": {
-    "@patternfly/patternfly": "^5.0.0",
-    "@patternfly/quickstarts": "^5.0.0-prerelease.2",
+    "@patternfly/patternfly": "^6.0.0-alpha.139",
+    "@patternfly/quickstarts": "^6.0.0-alpha.0",
     "@patternfly/transform-adoc": "*",
-    "@patternfly/react-core": "^5.0.0",
-    "asciidoctor": "^2.2.1",
+    "@patternfly/react-core": "6.0.0-alpha.50",
```

Comment:
I thought .61 was the alpha2 version of react-core?

Reply:
Ok, updated!